src/frontend/utils/converse.ts

```diff
@@ -41,6 +52,7 @@ export const converseMounter = () => {
           spoiler: false,
           toggle_occupants: false,
         },
+        whitelisted_plugins: ['marsha'],
```

Comment:
Maybe this could be a setting passed in appdata, or at least extract this list to settings.ts ?

Reply:
I don't see the interest to do that. `whitelisted_plugins` contains a list of external plugins you add to converse, they are added to configure converse, like other converse settings.

Reply:
It's just strange to me to find some harcoded values in code, but I get your point.